microcosm_flask/tests/conventions/test_crud.py

```diff
@@ -101,6 +101,29 @@ def test_create(self):
             },
         })
 
+    def test_create_empty_object(self):
+        response = self.client.post("/api/person", data='{}')
```

Comment:
This does not test exactly the use case we saw was breaking - however I could not for the life of me figure out how to reproduce a successful POST with 'empty' body similar to what httpie is doing..